### DISCUSSION TITLE
Add "first login" welcome / onboarding flow

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#3080] [Feature Requests]
- [ ] First Admin:
  - [ ] Project Setup (just first 4 fields)
  - [ ] User Profile Setup
  - [ ] Data Model: Explains process, button takes you to this page
- [ ] User: 
 -  [ ] User Profile Setup
  - [ ] “Project Onboarding Text”  (New Field in Settings)

**Comment by Nitwel** [comment]
Oh, thats an awesome Idea, love it. ❤️

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
